tensornetwork/tests/network_components_free_test.py

```diff
@@ -1457,7 +1457,7 @@ def test_edge_name_setter_throws_type_error(single_node_edge, name):
 
 def test_edge_node1_throws_value_error(single_node_edge):
   edge = Edge(node1=single_node_edge.node, axis1=0, name="edge")
-  edge._node1 = None
+  edge.node1 = None
```

Comment:
Should be `edge._nodes[0]` to match new design and `test_edge_node2_throws_value_error` below.

Reply:
Good point. Added.